templates/homepage.html

```diff
@@ -30,18 +30,20 @@
   </div>
 
   <div class="container">
-    <div class="row homepage-main-points">
+    <div class="row homepage-main-points"> style="text-align:center"
```

Comment:
That's not going to work, if anything, the style attribute should be inside the div. Then again, one should just use `text-center` as class.

Then again.. I don't really like centering the text in these columns.